src/components/StackedAvatar.tsx

```diff
@@ -1,11 +1,11 @@
 import Avatar from "./Avatar";
 
 export declare interface StackedAvatarProps {
-  contributors: DbContribution[];
+  contributors?: DbContribution[];
```

Comment:
making contributors optional in the prop is not correct, it will always be an empty array so it should not be omitted - what would an empty contributors array stacked avatar look like? 🤔